src/driver/memory.rs

```diff
@@ -0,0 +1,31 @@
+use url::Url;
+
+use driver::Driver;
+use error::Error;
+use query::{Range, SortRule, Condition, Query};
+use value::{Key, Iter};
+
+pub struct Memory;
```

Comment:
The memory driver. If no other driver is specified, this one is used. I added docs so hopefully that clarifies things.